### ISSUE TITLE
Allow a small delta when checking contrast ratio values

### ISSUE THREAD

**Issue by alice** [GoogleChrome/accessibility-developer-tools#44]
For example, 4.48 should be acceptable for WCAG's recommendation of 4.5 for normal text.

**Comment by alice** [comment]
This was fixed in baa436f48c8424512b1bfea45557412fe5598d0f.
